src/marqo/tensor_search/models/index_settings.py

```diff
@@ -250,6 +250,19 @@ def from_marqo_index(cls, marqo_index: core.MarqoIndex) -> "IndexSettings":
         else:
             raise api_exceptions.InternalError(f"Unknown index type: {type(marqo_index)}")
 
+    @classmethod
+    def get_model_properties(cls, marqo_index):
```

Comment:
@keshav , a quick question. Do we also hide this in Cloud control plane?

Reply:
Yes we already do, only Marqo is currently exposing it and this pr should fix it